CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 20.11.7
+* Added "setCustomMetrics" method to set the metrics you want to override or additional custom metrics you want to provide
```

Comment:
Remove " or additional custom metrics you want to provide", otherwise it will give the impression that it would work without a server side plugin